src/AutoRest.CSharp/Mgmt/AutoRest/MgmtConfiguration.cs

```diff
@@ -17,11 +17,22 @@ public class MgmtConfiguration
             string[] singletonResource,
             JsonElement? operationGroupToResourceType = default,
             JsonElement? operationGroupToResource = default,
-            JsonElement? operationGroupToParent = default)
+            JsonElement? operationGroupToParent = default,
+            JsonElement? mergeOperations = default)
```

Comment:
I don't see this config in the readme.md. Can we add some examples/tests for mergeOperations in the readme?

Reply:
It would take a while to extract the Deployments resource which uses mergeOperations from resources.json as I need to remove definitions already in Core. I will add the tests in a separate PR.

Reply:
Example added in readme of MgmtScopeResource.